ccmlib/common.py

```diff
@@ -927,18 +927,26 @@ def get_available_jdk_versions(env):
 def update_java_version(jvm_version=None, install_dir=None, cassandra_version=None, env=None,
                         for_build=False, info_message=None):
     """
-    Selects Java distribution and returns the updated env (updates JAVA_HOME and PATH variables if needed).
-    If 'jvm_version' is explicitly set, that one will be used if a Java distribution of that version can be found.
-    Otherwise, the Java version will be guessed from the distribution provided by 'install_dir' parameter or from
-    the provided 'cassandra_version' parameter.
-    :param jvm_version: The explicit Java version to use - must be the major Java version number like 8 or 11 (int).
+    Updates or fixes the environment variables for Java distribution.
+    If 'jvm_version' is explicitly set, that one will be used as long as it can be found in the current environment
+    via JAVA_HOME and JAVAx_HOME variables.
```

Comment:
```suggestion
    If 'jvm_version' is explicitly set, that one will be used as long as it can be found in the current environment via JAVA_HOME and JAVAx_HOME variables.
```